### PR TITLE
Datatype updates 121420a

### PR DESCRIPTION
**Pull request by jbranson000**
#1017 
Updating previously created data types with updated meta:tags for industry and replacing some measure datatypes with integers.

### CODE REVIEW

File: components/datatypes/file-transfer.schema.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          "description": "Type of the file transfer, if applicable."
+        },
+        "xdm:fileType": {
+          "title": "File Type",
+          "type": "string",
+          "description": "Type or category of file being up/downloaded by the visitor."
+        },
+        "xdm:fileURL": {
+          "title": "File URL",
+          "type": "string",
```
Comment (mjasrotia) [inline components/datatypes/file-transfer.schema.json:69]: format "uri" ?

File: components/datatypes/financial-account.schema.json [hunk 2]
```diff
@@ -33,9 +35,7 @@
         "xdm:financialAccountOwner": {
           "title": "Financial Account Owners",
           "type": "array",
-          "items": {
-            "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details"
-          },
+          "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details",
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:38]: Did you remove "items" here ?

Comment (mjasrotia) [reply components/datatypes/financial-account.schema.json]: This is still unresolved. We need to add "items" here

**Review by mjasrotia — APPROVED** [review]

File: components/datatypes/financial-account.schema.json [hunk 3]
```diff
@@ -10,14 +10,15 @@
...
   "meta:extensible": true,
-  "meta:industry": ["financial_services"],
   "description": "Financial account details, such as owners, balances, and types.",
   "definitions": {
     "financial-account": {
       "properties": {
         "xdm:currentAccountBalance": {
           "title": "Current Financial Account Balance",
-          "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+          "items": {
```
Comment (mjasrotia) [inline components/datatypes/financial-account.schema.json:19]: Is this supposed to be an array ? IF yes, we need to have a type too. Please clarify

File: components/datatypes/internal-search.schema.json [hunk 4]
```diff
@@ -0,0 +1,140 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/internal-search",
```
Comment (mjasrotia) [inline components/datatypes/internal-search.schema.json:8]: What makes this internal to be specific. Could this be a general search/site-search or something like that ?

File: components/datatypes/internal-search.schema.json [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+          "description": "The internal search term value with merchandising binding. Will show what term was searched for immediately before viewing a product."
+        },
+        "xdm:refineSearch": {
+          "title": "Refinement Value",
+          "type": "string",
+          "description": "Number of times an internal search refinement has occurred."
+        },
+        "xdm:refineType": {
+          "title": "Search Refinement Type",
+          "type": "array",
```
Comment (mjasrotia) [inline components/datatypes/internal-search.schema.json:39]: items missing

File: components/datatypes/financial-account.schema.json (hunk 3 above)
Comment (jbranson000) [reply components/datatypes/financial-account.schema.json]: It was not.  It is fixed now.

File: components/datatypes/financial-account.schema.json (hunk 2 above)
Comment (jbranson000) [reply components/datatypes/financial-account.schema.json]: I believe this is outdated and items does exist in the current schema.

File: components/datatypes/internal-search.schema.json (hunk 5 above)
Comment (jbranson000) [reply components/datatypes/internal-search.schema.json:39]: The intention of this field is for any search filter applied (i.e. brand, color, price, review rating, etc.) to be captured in a delimited list in an array.  Would there be better type to use for this?

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**